explorer/test/Test/Pos/Explorer/BListenerSpec.hs.unused

```diff
@@ -0,0 +1,140 @@
+-- | This module tests BListener from 'Pos.Explorer'.
```

Comment:
Why do we need this file?

Reply:
It's here to test `BListener`, but it will be added once I merge the tests for _explorer_. There is a comment on the PR. Also, there is a comment on `cardano-sl-explorer.cabal` - https://github.com/input-output-hk/cardano-sl/pull/1876/files#diff-71bed27630919bce85c19d701a88c3f3R341